swagger.yaml

```diff
@@ -8,6 +8,59 @@ security:
   - bearerAuth: []
 
 paths:
+  "/users/{userId}/vacation":
+    put:
+      operationId: updateUserVacation
+      summary: Update user vacation days
+      description: Update total and remaining vacation days for a specific user by year.
+      tags:
+        - Update vacation
+      parameters:
+        - name: userId
+          in: path
+          required: true
+          description: The Keycloak ID of the user to update vacation for
+          schema:
+            type: string
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                vacationDays:
+                  type: object
+                  additionalProperties:
+                    type: object
+                    properties:
+                      total:
+                        type: integer
+                      remaining:
+                        type: integer
+                  description: Vacation days by year
+              required:
+                - vacationDays
+      responses:
+  "200":
+    description: Successfully updated the vacation days
+    content:
+      application/json;charset=utf-8:
+        schema:
+          type: object
+          properties:
+            id:
+              type: string
+              description: The Keycloak ID of the updated user
+            vacationDays:
+              type: object
+              additionalProperties:
+                type: object
+                properties:
+                  total:
+                    type: integer
+                  remaining:
+                    type: integer
+                    
```

Comment:
You should still have the generic error resposne e.g.:
 "default":
          description: Invalid request was sent to the server.
          content:
            application/json:
              schema:
                $ref: "#/components/schemas/Error"